querybook/webapp/ui/Tag/Tag.tsx

```diff
@@ -27,16 +26,42 @@ export interface ITagProps {
 
     className?: string;
 }
-export interface IHoverIconTagProps extends ITagProps {
-    iconOnHover?: AllLucideIconNames;
-    onIconHoverClick?: () => any;
-}
 
 export const TagGroup: React.FunctionComponent<ITagGroupProps> = ({
     className,
     children,
 }) => <div className={`${className} TagGroup`}>{children}</div>;
 
+const StyledColorTag = styled.span.attrs<{
+    highlighted?: boolean;
+    light?: boolean;
+    color?: string;
```

Comment:
are we allowing all colors? i dont think we should - lets limit to the colors we have red, blue, yellow, pink, green, etc. and use the -light or -lightest for bg and -dark or -darkest for text